docs/conf.py

```diff
@@ -46,14 +44,12 @@
 try:
     import suds
 except ImportError:
-    raise ImportError('suds could not be imported, please install the '
-                      '"suds-jerko" package and try again')
+    print('ERROR: the documentation requires the suds-jerko package to be installed')
```

Comment:
I preferred the wording of the previous error, because it's explicitly calling out the difference between `suds` and `suds-jurko`.